.env.local.php.example

```diff
@@ -22,4 +22,6 @@ return [
     ],
     // The tag PHID that indicates whether a task is in review
     'REVIEW_TAG_PHID' => '',
+    // comma separated names of Phabricator user names that are allowed to create projects on Phragile
+    'PHRAGILE_ADMINS' => ''
```

Comment:
To be honest I am not sure what the best practice is in this case. There is a class to handle configuration values using `Config::set` and `Config::get` which gets its information from the files in `app/config`. This seemed redundant to me since I would have to put the actual value in `.env.local.php` and then retrieve that value again using something like `Config.set('PHRAGILE_ADMINS', $_ENV['PHRAGILE_ADMINS'])` somewhere in `app/config`.
So I thought I might as well just put them here and retrieve them with `$_ENV`.

Using the Phabricator names there seemed the easiest way to do this for now without implementing actual user roles just for this one functionality. Arrays aren't supported for some reason so I went with a comma separated string.